libs/core/src/lib/calendar/calendar-header/calendar-header.component.ts

```diff
@@ -184,16 +179,12 @@ export class CalendarHeaderComponent<D> implements OnDestroy, OnInit, OnChanges
     }
 
     /** @hidden */
-    private _listenToLocaleChanges(): void {
+    private _listenToCalendarLabelsChanges(): void {
+
         this._dateTimeAdapter.localeChanges.pipe(takeUntil(this._onDestroy$)).subscribe(() => {
-            this._calculateMonthNames();
             this._calculateLabels();
-            this._changeDetRef.markForCheck();
```

Comment:
![mark_for_check](https://user-images.githubusercontent.com/5276786/118147378-2bae7380-b418-11eb-840e-ca1b205717a7.png)
labels was removed, right now we have 2 calls mark for check - both from you.
without it - labels not works (in case of non stream labels)